db/migrations/setting_up_db.sql

```diff
@@ -0,0 +1,20 @@
+CREATE DATABASE chitter;
+
+\c chitter
+
+Create Users table
```

Comment:
these headers are actually a little confusing and I ended up accidentally putting them in my psql by accident as vs code colour codes them similarly